Makefile

```diff
@@ -131,35 +133,63 @@ defconfig-help:
 		done
 	@echo ""
 
-.PHONY: i915dkmsdeb-pkg-help
-i915dkmsdeb-pkg-help:
-	$(info Debian package contains the default kernel version (KV of UBUNTU_OEM_22.04))
-	$(info To create the package with specific kernel version, \
-		pass the supported kernel name to OS_DISTRIBUTION option)
-	$(info )
-	$(info ##### List of supported osv kernel versions #####)
-	$(shell cat versions |& tail -n +4 | cut -d "_" -f 1-3 | grep "UBUNTU" | grep -v "GENERIC_KERNEL" 1>&2)
-	$(shell cat versions |& tail -n +4 | cut -d "_" -f 1-2 | grep VANILLA_5.15LTS 1>&2)
-	$(info )
-	$(info Example: make i915dkmsdeb-pkg OS_DISTRIBUTION=UBUNTU_22.04_GENERIC)
+.PHONY: common-help
+common-help:
+	@echo "Build Configurations:"
+	@echo "  KLIB   : path/to/headers"
+	@echo "  KLIB_BUILD     : path/to/headers/build "
+	@echo "  BUILD_VERSION  : Pass build version to be added to package name"
+	@echo "  RELEASE_TYPE   : <opensource/prerelease/custom> Package release type created"
+	@echo " 			Example: make <Target> RELEASE_TYPE=test "
+	@echo " 			Package names would be intel-i915-dkms-test for DKMS or intel-i915-test for binary"
+	@echo " 			Note: If custom packages are created, tracking the conflicting package "
+	@echo " 			is difficult. Make sure no other package is already installed before "
+	@echo " 			you intalling current one."
+	@echo "  BUILD_CONFIG   : Specifiv build config variant"
+	@echo " 			Ex: make <Target> BUILD_CONFIG=sp2 "
+	@echo "  OS_DISTRIBUTION: Distro targeted package"
```

Comment:
BUILD_CONFIG = sp2 , its not supported OS now

Reply:
removed.